yilan_zhou/week_01/week1-4_bank.js

```diff
@@ -0,0 +1,96 @@
+//1/10/2015  week1-4 homeowork
+//https://gist.github.com/wofockham/dacf2da17c743afb2b17
+//js bank
+
+var bank = [  //bank array with name and account
+    {name: 'Mary',
+     balance: 100
+    },{
+     name: 'David',
+     balance: 200
+    },{
+     name: 'Jane',
+     balance:300
+    }];
+
+function sumBankBalance(thisBank) {   
+    var total = 0;
+    for (var i = 0; i<thisBank.length;i++) {
+       total += thisBank[i].balance; 
+    }
+    return "Total Bank Balance: " + total;
+}
+
+function newBankAccount(thisBank,newName,newBalance) {
```

Comment:
Good job, by taking bank as an argument you make your newBankAccount very flexible, so you can add accounts to different banks! :+1: